django-stubs/contrib/admin/options.pyi

```diff
@@ -265,7 +265,7 @@ class ModelAdmin(BaseModelAdmin[_ModelT]):
     def changelist_view(self, request: HttpRequest, extra_context: dict[str, Any] | None = ...) -> HttpResponse: ...
     def get_deleted_objects(
         self, objs: Sequence[_ModelT] | QuerySet[_ModelT], request: HttpRequest
-    ) -> tuple[list[Model], dict[str, int], set[str], list[str]]: ...
+    ) -> tuple[list[str], dict[str, int], set[str], list[str]]: ...
```

Comment:
Thanks! Looks like the related `get_deleted_objects` function in `contrib/admin/utils.pyi` should be updated too.

Reply:
Fixed in [c99d322](https://github.com/typeddjango/django-stubs/pull/1825/commits/c99d322874b5b74157f681232c4471f8481779e9)